### PR TITLE
Use potions and whetstones in between battles

### PR DESCRIPTION
**Pull request by PiazzaGio**
For issue [#8](https://github.com/tagniam/Turn/issues/8)

### CODE REVIEW

**Comment by PiazzaGio** [comment]
I did three commits because I fixed the indentation since VS broke it.

File: Turn/src/Player.cpp [hunk 1]
```diff
@@ -122,6 +122,53 @@ int Player::Attack(){
...
+		switch (choice) {
+		case 0:
+			return;
+		case 1:
+			// Player drinks a potion.
+			// Does not execute if there are no potions in the inventory.
+			if (potions > 0) {
+				UsePotion();
+				Sleep(SLEEP_MS);
+			}
```
Comment (tagniam) [inline Turn/src/Player.cpp]: If the player has 0 potions there should be a message displayed here, followed by `Sleep(SLEEP_MS)`

File: Turn/src/Player.cpp [hunk 2]
```diff
@@ -122,6 +122,53 @@ int Player::Attack(){
...
+
+			break;
+		case 2:
+			// Player sharpens their weapon with a whetstone.
+			// Does not execute if there are no whetstones in inventory.
+			// No damage is done to the enemy.
+			if (whetstones > 0) {
+				UseWhetstone();
+				Sleep(SLEEP_MS);
+			}
```
Comment (tagniam) [inline Turn/src/Player.cpp]: Same thing as above

**Comment by PiazzaGio** [comment]
Added the message when the player doesn't have enough potions/whetstones

**Comment by tagniam** [comment]
All good, thanks @PiazzaGio 👍

**Pull request merged**